README.md

```diff
@@ -1 +1,151 @@
-# leaderboard
\ No newline at end of file
+
+          Leaderboard
+
+<a name="readme-top"></a>
+<div align="center">
+  <h3><b>Leaderboard </b></h3>
+</div>
+<!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Deployment](#deployment)
```

Comment:
- [ ] Kindly remove this link to the `Deployment` subsection as this part has been removed making this link broken